Core/MigrationOptions.cs

```diff
@@ -23,14 +23,14 @@ namespace NUnit2To3SyntaxConverter
 {
   public class MigrationOptions
   {
-    private static readonly Encoding DefaultEncoding = Encoding.Default;
+    private static readonly Encoding s_defaultEncoding = Encoding.Default;

```

Comment:
Actually, I don't think this field is necessary. You could just directly assign to the autoproperty